source/features/close-out-of-view-modals.tsx

```diff
@@ -29,9 +29,9 @@ function menuActivatedHandler(event: delegate.Event): void {
 	const details = event.target as HTMLDetailsElement;
 
 	// Safety check #3742
-	if (!details.open && lastOpen > Date.now() - 1000) {
+	if (!details.open && lastOpen > Date.now() - 500) {
 		delegation!.destroy();
-		features.error(__filebasename, 'Modal was closed too quickly. Disabling feature');
+		console.warn(`The modal was closed too quickly. Disabling ${__filebasename} for this session.`);
```

Comment:
## Reduced error for `close-out-of-view-modals`

It would show a red error if you were fast enough. Now the timeout is reduced and it just shows a warning.

<img width="938" alt="Screen Shot 10" src="https://user-images.githubusercontent.com/1402241/116376327-560a0980-a83a-11eb-9c21-a8bb91602ffc.png">

Reply:
I doubt it because this event is always generated by the browser regardless of the origin of the click, which has a separate event